src/sentry/api/endpoints/user_password.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+
+from django.utils.crypto import constant_time_compare
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.bases.user import UserEndpoint
+from sentry.api.decorators import sudo_required
+from sentry.auth import password_validation
+from sentry.models import User
+from sentry.security import capture_security_activity
+
+
+class UserPasswordSerializer(serializers.ModelSerializer):
+    passwordVerify = serializers.CharField(max_length=128, required=True)
+    passwordNew = serializers.CharField(max_length=128, required=True)
+    password = serializers.CharField(max_length=128, required=True)
+
+    class Meta:
+        model = User
+        fields = ('password', 'passwordNew', 'passwordVerify', )
+
+    def validate_password(self, attrs, source):
```

Comment:
mm don't you want this to be `validate_passwordNew`? otherwise i think you're just validating their old password.

Reply:
oops, good catch! `password` used to be new password

Reply:
Fixed + added a test for this